compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala

```diff
@@ -41,8 +47,10 @@ case class ApplicationCompiler(typeAliases: Map[String, Type],
                                parameterLinkSerializer: ParameterLinkSerializer,
                                useManifests: Boolean,
                                dxApi: DxApi = DxApi.get,
-                               logger: Logger = Logger.get)
-    extends ExecutableCompiler(extras, parameterLinkSerializer, dxApi) {
+                               logger: Logger = Logger.get,
+                               project: DxProject,
+                               folder: String)
+    extends ExecutableCompiler(extras, parameterLinkSerializer, dxApi, project, folder) {
```

Comment:
It looks like project and folder are not used in the superclass, so they do not need to be passed to the superclass constructor.

Reply:
For now in this PR I can make it apply only to ApplicationCompiler (if cloning at all).